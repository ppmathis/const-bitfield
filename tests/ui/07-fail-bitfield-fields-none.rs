#![feature(const_mut_refs)]
#![feature(const_trait_impl)]

use const_bitfield::bitfield;

bitfield! {
    struct BitNone(u8);
    bool, _, _: 0;
}

bitfield! {
    struct BitsNone(u8);
    u8, _, _: 0;
}

pub fn main() {}
