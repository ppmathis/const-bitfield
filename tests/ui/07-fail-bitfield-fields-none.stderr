error: no rules expected the token `field`
  --> tests/ui/07-fail-bitfield-fields-none.rs:7:1
   |
7  | / bitfield! {
8  | |     struct BitNone(u8);
9  | |     bool, _, _: 0;
10 | | }
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::bitfield` (in Nightly builds, run with -Z macro-backtrace for more info)

error: no rules expected the token `field`
  --> tests/ui/07-fail-bitfield-fields-none.rs:12:1
   |
12 | / bitfield! {
13 | |     struct BitsNone(u8);
14 | |     u8, _, _: 0;
15 | | }
   | |_^ no rules expected this token in macro call
   |
   = note: this error originates in the macro `$crate::bitfield` (in Nightly builds, run with -Z macro-backtrace for more info)
